}());} catch (e) {
  try {
    console.log(e);
  } catch (ex) {}
}